LEAF_Request_Portal/templates/custom_override/view_homepage.tpl

```diff
@@ -0,0 +1,48 @@
+<div class="menu2" style="width: 315px; float: left">
+
+<a href="?a=newform" role="button">
+    <span class="menuButtonSmall" style="background-color: #2372b0; color: white">
+        <img class="menuIconSmall" src="../libs/dynicons/?img=document-new.svg&amp;w=76" style="position: relative" />
+        <span class="menuTextSmall" style="color: white">New Request</span><br />
+        <span class="menuDescSmall" style="color: white">Start a new request</span>
+    </span>
+</a>
+    
+
+<!--{if $inbox_status == 0}-->
+<a href="?a=inbox" role="button">
+    <span class="menuButtonSmall" style="background-color: #c9c9c9">
+        <img class="menuIconSmall" src="../libs/dynicons/?img=folder-open.svg&amp;w=76" style="position: relative" />
```

Comment:
This template is not under the admin folder. what is the use?

Reply:
Don't know how/why that got changed, I didn't edit that one directly.

Reply:
When I pulled the branch the file is not there. Need to investigate how that file got created/edited...